docs/src/man/getting_started.md

```diff
@@ -351,6 +365,71 @@ we can observe that:
 * the third column `:c` can hold categorical data; here we notice `…`, which indicates that the actual name of the type was long and got truncated;
 * the type information in fourth column `:d` presents a situation where the name is both truncated and the type allows `Missing`.
 
+#### Referencing Single Cells
+
+Now that your DataFrame is created, you can reference single cells by row number and column name. You can use the DataFrame.name notation, or the :name notation to refer to columns.
```

Comment:
How to refer to whole columns was described in the section above, do you think it is worth to repeat this information again?